workflows/check.yml

```diff
@@ -0,0 +1,20 @@
+name: check
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install prereq
+      run: |
+        brew install docker tfenv tflint
+        tfenv install
+
+    - name: tf fmt
+      run: |
+        terraform fmt
+    - name: tflint
+      run: |
+        tflint
```

Comment:
do we still want to be running `tflint` if we're taking it out of the pre-commit hook?

Reply:
I figured yes, because it seems to run successfully in the GH action.

Reply:
If it works it works